docs/querying/sorting-orders.md

```diff
@@ -30,7 +30,7 @@ title: "String comparators"
 These sorting orders are used by the [TopNMetricSpec](./topnmetricspec.md), [SearchQuery](./searchquery.md), GroupByQuery's [LimitSpec](./limitspec.md), and [BoundFilter](./filters.md#bound-filter).
 
 ## Lexicographic
-Sorts values by converting Strings to their UTF-8 byte array representations and comparing lexicographically, byte-by-byte.
```

Comment:
This one seems correct to me and need not be changed.

Reply:
Thank you very much for your help in reviewing this pr! I will revert this change.